api/package.json

```diff
@@ -5,7 +5,7 @@
   "description": "API for the vercel/vercel repo",
   "main": "index.js",
   "scripts": {
-    "build": "yarn --cwd .. && node ../utils/run.js build all"
+    "vercel-build": "yarn --cwd .. && node ../utils/run.js build all"
```

Comment:
I think the API should probably strip the new properties so it remains unchanged. Compare your PR deployment to https://vercel.com/api/v1/frameworks and you'll see the diff.

Reply:
8830dd9f559685a9b42156629dd2a90d5495391c